o2a/converter/oozie_converter.py

```diff
@@ -111,14 +111,17 @@ def convert_nodes(self):
         """
         logging.info("Converting nodes to tasks and inner relations")
         for p_node in self.workflow.nodes.values():
+            if p_node.tasks:
```

Comment:
Because I need to create a node with a pre-defined list of tasks. Create tasks that are not created by mappers, but by the converter. This means that there is no attempt to run the mapper, which this node does not have. Now it is possible to create a node with predefined tasks.

Reply:
Maybe a "cleaner" approach would be to instead implement a `to_tasks_and_relations()` in the base mapper which returns empty dicts?